crates/prefab/src/save.rs

```diff
@@ -78,7 +78,10 @@ pub enum SaveState {
     Idle,
 }
 
-fn prepare_children(mut commands: Commands, query: Query<(Entity, &Children), With<PrefabMarker>>) {
+fn prepare_children(
+    mut commands: Commands,
+    query: Query<(Entity, &Children), (With<PrefabMarker>, Without<SceneAutoChild>)>,
```

Comment:
it would be nice to add a test for prepare children that doesnt execute when it has SceneAutoCHild. There is already a test for the without